.github/ISSUE_TEMPLATE/ad-hoc-meeting.yaml

```diff
@@ -0,0 +1,53 @@
+name: 'Create an Ad-Hoc Meeting'
+description: 'This Issue template is used for creating a new Ad-Hoc Meeting for the Cross Project Council'
```

Comment:
We haven't used this template in years at this point. Do we actually need it?

Reply:
If we need or not, we can debate about that maybe on a separate issue? I'm fine over here too, but I want to avoid delaying this PR because of off-topic discussion xD